app/src/main/java/nl/avans/drivioapp/service/AdvertisementService.kt

```diff
@@ -1,25 +1,25 @@
-package nl.avans.drivioapp.service
-
-import nl.avans.drivioapp.model.Advertisement
-import retrofit2.Response
-import retrofit2.http.Body
-import retrofit2.http.GET
-import retrofit2.http.POST
-import retrofit2.http.Path
-
-interface AdvertisementService {
-    @GET("advertisement")
-    suspend fun getAdvertisements(): List<Advertisement>
-
-    @GET("advertisement/{advertisementId}")
-    suspend fun getAdvertisementById(
-        @Path(
-            value = "advertisementId",
-            encoded = false
-        ) advertisementId: Int
-    ): Response<Advertisement>
-
-    @POST(value = "advertisement")
-    suspend fun postAdvertisementWithResponse(@Body advertisement: Advertisement): Response<Unit>
-
-}
\ No newline at end of file
+package nl.avans.drivioapp.service
+
+import nl.avans.drivioapp.model.Advertisement
+import retrofit2.Response
+import retrofit2.http.Body
+import retrofit2.http.GET
+import retrofit2.http.POST
+import retrofit2.http.Path
+
+interface AdvertisementService {
+    @GET("advertisement")
+    suspend fun getAdvertisements(): List<Advertisement>
+
+    @GET("advertisement/{advertisementId}")
+    suspend fun getAdvertisementById(
+        @Path(
+            value = "advertisementId",
+            encoded = false
+        ) advertisementId: Int
+    ): Response<Advertisement>
+
+    @POST(value = "advertisement")
+    suspend fun postAdvertisementWithResponse(@Body advertisement: Advertisement): Response<Unit>
+
+}

```

Comment:
Ik hoop dat dit niet gaat conflicten met het repository pattern die is ingebouwd.

Reply:
Hij geeft hier geen conflict op, ik ga er dus vanuit dat git het goed oppakt

Reply:
Check!